docs/src/examples/02b-model-modifications.jl

```diff
@@ -163,5 +171,22 @@ flux_changes =
 # ...and again see what changed most:
 sort(collect(flux_changes), by = last)
 
-#md # !!! tip "For realistic comparisons always use a uniquely defined flux solution"
-#md #     Since the usual flux balance allows a lot of freedom in the "solved" flux and the only value that is "reproducible" by the analysis is the objective, one should never compare the flux distributions directly. Typically, that may result in false-positive (and sometimes false-negative) differences. Use e.g. [parsimonious FBA](03-parsimonious-flux-balance.md) to obtain uniquely determined and safely comparable flux solutions.
+#md # !!! tip "Tip: For realistic comparisons always use a uniquely defined flux solution"
+#md #    Since the usual flux balance allows a lot of freedom in the "solved" flux and 
```

Comment:
I was keeping the admonitions on a single line on a purpose, because indenting these properly is a pita. notice that you don't have the required 4 indent spaces here.

Reply:
aahhhh